expected/add_agg.out

```diff
@@ -11,7 +11,7 @@ DROP TABLE IF EXISTS test_khvengxf;
 NOTICE:  table "test_khvengxf" does not exist, skipping
 CREATE TABLE test_khvengxf (
 	val    integer
-);
+) DISTRIBUTED RANDOMLY;
```

Comment:
Is there any specific reason that we have to force randomly here?

Reply:
No, there isn't any specific reason. I just set the random distribution policy as default when creating tables in this commit: https://github.com/greenplum-db/postgresql-hll/pull/2/commits/38838325f0ae0a2231f8a971984562462f0679cf. Maybe this will cover some MPP cases?